bin/wp-env-config.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+## Check if user already exists
+wp user get customer 2> /dev/null
+
+## if 0 is the exit code then we can leave otherwise we'll try creating the user
+if [ $? -eq 0 ]
+then
+	EXIT_CODE=0
+else
+  wp user create customer customer@woocommercecoree2etestsuite.com --user_pass=password --role=customer
+  EXIT_CODE=$?
+fi
+
+## set permalinks for easier wp-json
+wp rewrite structure '/%postname%/'
+wp plugin list
```

Comment:
this should be deleted.

Reply:
why? Permalinks is necessary and `wp plugin list` helps verify the plugins that were installed and activated.

Reply:
I was referring to `wp plugin list`, we only used to test if the plugin was installed, if you think we need it for future reference, we can keep it.